api/graphql/makeModels.js

```diff
@@ -126,7 +128,7 @@ export default function makeModels (userId, isAdmin) {
         'followers',
         'linkPreview'
       ],
-      filter: nonAdminFilter(sharedPostMembership('posts', userId)),
+      filter: nonAdminFilter(composeFilters(activePost, sharedPostMembership)('posts', userId)),
```

Comment:
The `tableName, userId, q` signature is pretty specific to the membership filters. It would make more sense to me to see something like:

```js
composeFilters(
  activePost,
  nonAdminFilter(sharedPostMembership('posts', userId))
)
```